packages/rulesets/docs/sharing-rulesets.md

```diff
@@ -38,7 +38,7 @@ extends:
   - https://raw.githubusercontent.com/openapi-contrib/style-guides/master/apisyouwonthate.yml
 ```
 
-As with any ruleset, you can pass these directly to the [Spectral CLI](./2-cli.md):
+As with any ruleset, you can pass these directly to the [Spectral CLI](../../cli/README.md):
```

Comment:
will this work when the package is published to NPM?
I doubt it'll - perhaps we link to meta.stoplight.io?